### DISCUSSION TITLE
Firefox error when calling d3.pointer inside of 'zoom' event handler

### DISCUSSION THREAD

**Discussion by tsairinius** [d3/d3#3907] [Q&A] [answered]
In the following example, I am calling d3.pointer inside of a 'zoom' event handler. The zoom behavior is being applied to an SVG element. When I call the zoom behavior's `scaleTo` method, the following error occurs in Firefox (Chrome and Edge seem to work fine):

"TypeError: SVGPoint.x setter: Value being assigned is not a finite floating-point value." 

I've also tried replacing `scaleTo` with `scaleBy` and `transform`, but the same error pops up. 

See example here: https://jsfiddle.net/nLt523mj/

**Comment by Fil** [comment, marked as answer]
thanks for the report; let's track this at https://github.com/d3/d3-selection/issues/319
